packages/sdk-utils/src/post-screenshot.js

```diff
@@ -0,0 +1,18 @@
+import percy from './percy-info.js';
+import request from './request.js';
+
+// Post screenshot data to the CLI automateScreenshot endpoint. If the endpoint responds with a build error,
+// indicate that Percy has been disabled.
+export async function captureAutomateScreenshot(options, params) {
+  let query = params ? `?${new URLSearchParams(params)}` : '';
```

Comment:
Can we use a default value for params here instead?

Reply:
it is copied from https://github.com/percy/cli/blob/5f86910347e74cb9125ec8a5fb7d90155af44b04/packages/sdk-utils/src/post-snapshot.js#L7